src/amo/components/HeadMetaTags/index.js

```diff
@@ -0,0 +1,88 @@
+/* @flow */
+import config from 'config';
+import * as React from 'react';
+import Helmet from 'react-helmet';
+import { connect } from 'react-redux';
+import { compose } from 'redux';
+
+import { getCanonicalURL } from 'amo/utils';
+import type { AppState } from 'amo/store';
+
+type Props = {|
+  date?: Date | null,
```

Comment:
@kumar303 suggested in https://github.com/mozilla/addons-frontend/pull/6806#discussion_r231566445 that we should avoid using `?` in Flow property definitions. Do you agree? Do you think they should therefore not be used here?